src/mpi/init/initthread.c

```diff
@@ -324,6 +324,15 @@ int MPIR_Init_thread(int * argc, char ***argv, int required, int * provided)
     int exit_init_cs_on_failure = 0;
     MPIR_Info *info_ptr;
 
+#if (MPL_THREAD_PACKAGE_NAME == MPL_THREAD_PACKAGE_ARGOBOTS)
+    int rc = ABT_init(*argc, *argv);
```

Comment:
IMO, the user is the one responsible to initialize Argobots, not MPICH. We could, however, check that Argobots is initialized with `ABT_initialized`
@pavanbalaji , do you agree on this?